src/GameLoop.cpp

```diff
@@ -301,16 +315,15 @@ void GameLoop::RunGameLoop()
 
 		UpdateAllButDraw(false);
 
-		if( INPUTMAN->DevicesChanged() )
+		SCREENMAN->Draw();
```

Comment:
Is there a functional difference between moving the Draw before InputDriverCheck?

Reply:
No, everything keeps working as expected. I haven't been able to find a reason it was done in this order. It seems to me like it would be best to update the screen and then check for input device changes and I haven't observed any changes in game behavior by doing this.

Reply:
Let's just keep the original ordering for now to isolate the changes. We can change the ordering separately.

Reply:
Okay, lemme swap those around quickly and i'll squash the history here.

Reply:
https://github.com/itgmania/itgmania/pull/276/commits/927521633979f474f26fc3297e4b4c8f8f99f26d resolves this.